front-end/src/app/transactions/transaction-group-ag/transaction-group-ag.component.ts

```diff
@@ -44,7 +45,7 @@ export class TransactionGroupAgComponent extends TransactionTypeBaseComponent im
 
   bForm: FormGroup = this.fb.group({});
 
-  bValidateService: ValidateService;
+  // bValidateService: ValidateService;
```

Comment:
This was failing the typescript check of needing undefined added to its type so commented out so author could fix.

Reply:
This will be corrected in #459